llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -19227,6 +19227,16 @@ bool RISCV::CC_RISCV(const DataLayout &DL, RISCVABI::ABI ABI, unsigned ValNo,
 
   ArrayRef<MCPhysReg> ArgGPRs = RISCV::getArgGPRs(ABI);
 
+  const RISCVSubtarget &STI =
+      State.getMachineFunction().getSubtarget<RISCVSubtarget>();
```

Comment:
Use `TLI.getSubtarget()`?

Reply:
Oh I see your another PR https://github.com/llvm/llvm-project/pull/107503, you will remove `TLI`.